test/frontend/test_node.py

```diff
@@ -86,6 +87,12 @@ def do_test_expected(self):
       with tf.Session() as sess:
         tf_output = sess.run(test_op, tf_feed_dict)
 
+      # make sure backend_output and tf_output are Iterable
+      if backend_output.ndim == 0:
+        backend_output = backend_output.reshape(1)
+      if isinstance(tf_output, Iterable) == False:
+        tf_output = [tf_output]
+
```

Comment:
Why do we need this block?

Reply:
What is the reason that error occurs after this patch? I mean before this, every think works well.

Reply:
It is because for size(), backend_output.shape could be (), and tf_output type could be <class 'numpy.int32'>. Neither is iterable.